test/e2e/data/infrastructure-aws/kustomize_sources/external-vpc-clusterclass/byo-infra-variables.yaml

```diff
@@ -0,0 +1,28 @@
+- op: add
```

Comment:
So... (and this is probably my lack of knowledge in this area, but please bare with me) They way these different from currently configurable networking, like just plain adding your VPC ID into the cluster config, is that these values will now be managed fields and used in SSA. So the user doesn't have to change things in a config file, but rather apply new patch values? Did I get that right?

Reply:
Yes, this particular scenario tests BYO VPC and subnets which was not working before SSA and id field being compulsory.